tests/gl_draw_elements.c

```diff
@@ -0,0 +1,53 @@
+#include <GL/gl.h>
+#include <GL/glut.h>
+
+typedef struct Vertex {
+    short           x, y, z;
+    unsigned char   c_r, c_g, c_b, c_a;
+    short           u,v;
+} Vertex;
+
+Vertex triangle[3] = {
+    0, 0, 0,   0, 255, 255, 255, 0, 0,
+    1, 1, 0, 255,   0, 255, 255, 0, 0,
+    1, 0, 0, 255, 255,   0, 255, 0, 0
+};
+unsigned short indices[3] = { 0, 1, 2};
+
+void draw() {
+    glMatrixMode(GL_PROJECTION);
+    glLoadIdentity();
+
+    glMatrixMode(GL_MODELVIEW);
+    glLoadIdentity();
+
+    glClearColor(1, 1, 0, 1);
+    glClear(GL_COLOR_BUFFER_BIT);
+
+    glEnableClientState(GL_VERTEX_ARRAY);
+    glEnableClientState(GL_COLOR_ARRAY);
```

Comment:
Old fixed pipeline GL has already begun to escape my brain, I wonder if this needs a `glEnableClientState(GL_TEXTURE_COORD_ARRAY)` as well?